.github/workflows/build-test.yml

```diff
@@ -201,4 +201,25 @@ jobs:
       - name: Build with Maven
         run: |
           mvn clean install -pl -:gremlin-javascript,-:gremlin-python,-gremlin-dotnet,-:gremlin-dotnet-source,-:gremlin-dotnet-tests -q -DskipTests -Dci
-          mvn verify -pl :neo4j-gremlin -DincludeNeo4j
\ No newline at end of file
+          mvn verify -pl :neo4j-gremlin -DincludeNeo4j
+  go:
+    name: Go
+    timeout-minutes: 20
+    needs: smoke
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK11
```

Comment:
Do we need to build it specifically with maven from GitHub actions? Can we use the native Go tools?

Reply:
Rithin and I talked, we iwll use native Go tools for now. 
Plan:
1. Launch tinkerpop gremlin server through docker
2. Build gremlin go driver
3. Run gremlin go driver tests
4. Shut down tinkerpop gremlin server through docker